spark-bigquery-dsv1/spark-bigquery-dsv1-parent/pom.xml

```diff
@@ -36,7 +36,7 @@
     </dependency>
     <dependency>
       <groupId>${project.groupId}</groupId>
-      <artifactId>spark-bigquery-pushdown_${scala.binary.version}</artifactId>
+      <artifactId>spark-bigquery-pushdown-common_${scala.binary.version}</artifactId>
```

Comment:
dsv1 modules now depend on `spark-bigquery-pushdown-common_${scala.binary.version}`.

`spark-bigquery-pushdown-common_2.12` will contain the common query pushdown code

`spark-bigquery-pushdown-common_2.11` does not contain any pushdown code. Instead it implements a couple of util methods that throw a `NotImplementedError`